짝수_홀수_찾기.py

```diff
@@ -0,0 +1,7 @@
+def solution(num):
+    if(num%2 == 0):
+        answer = "Even"
+    else:
+        answer = "Odd"
+    
+    return answer
```

Comment:
```suggestion
    return "Even" if (num % 2 == 0) else "Odd"
```
이렇게 한 줄로도 할 수 있어요 삼항연산자에 대해서 한번 찾아보고 공부해보세요